R/model_diagnostics.R

```diff
@@ -45,9 +46,9 @@ model_diagnostics <-  function(explainer, variables = NULL, ...) {
   # if variables = NULL then all variables are added
   # otherwise only selected
   if (is.null(variables)) {
-    results <- explainer$data
+    results <- as.data.frame(explainer$data)
```

Comment:
do we need to covert it to data.frame?
would it be simple to use `rbind()` to keep the class of an object?

Reply:
At first, I used `cbind` in 10 places to add these columns. But then, don't we want the result of the explanation to be `data.frame`? It is used by plot etc.